.github/workflows/test-db-ops.yaml

```diff
@@ -132,36 +103,23 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}-ee
           tags: |
-            type=raw,value=${{ github.event_name == 'repository_dispatch' && github.event.client_payload.ref || inputs.environment }}
-            type=raw,value=${{ github.event_name == 'repository_dispatch' && steps.lastcommit.outputs.shorthash || inputs.sha }}-${{ github.event_name == 'repository_dispatch' && github.event.client_payload.ref || inputs.environment }}
-            type=raw,value=latest,enable=${{ github.event_name == 'repository_dispatch' && github.event.client_payload.ref  == 'prod' ||  inputs.environment == 'prod' }}
+            type=raw,value=${{ env.MIGRATION_ENV }}
+            type=raw,value=${{ env.SHORT_HASH }}-${{ steps.lastcommit.outputs.shorthash }}-${{ env.MIGRATION_ENV }}
```

Comment:
Please add comment that explain this line

<WF SHA>-<Migration SHA>-<ENV Name | Branch Name>